Add CI check for missing dependencies in package

Are you experiencing an issue with...

- [ ] [shields.io](https://shields.io/#/)
- [ ] My own instance
- [x] [badge-maker NPM package](https://www.npmjs.com/package/badge-maker)

:beetle: **Description**

**badge-maker version: 3.0.0**

when consuming the "badge-maker" as a dependency the dependency "camelcase" is not installed and it causes a failure:

> module.js:550
>     throw err;
>     ^
> 
> Error: Cannot find module 'camelcase'
>     at Function.Module._resolveFilename (module.js:548:15)
>     at Function.Module._load (module.js:475:25)
>     at Module.require (module.js:597:17)
>     at require (internal/module.js:11:18)


:bulb: **Possible Solution**

<!--- Optional: only if you have suggestions on a fix/reason for the bug -->

**add to the [dependencies list in package.json](https://github.com/badges/shields/blob/master/badge-maker/package.json#L36)
camelcase: "5.x.x"**

--- a/badge-maker/CHANGELOG.md
+++ b/badge-maker/CHANGELOG.md
@@ -1,5 +1,9 @@
 # Changelog
 
+## 3.0.1
+
+- Fix missing dependency
+
 ## 3.0.0
 
 ### Breaking Changes

--- a/badge-maker/package.json
+++ b/badge-maker/package.json
@@ -1,6 +1,6 @@
 {
   "name": "badge-maker",
-  "version": "3.0.0",
+  "version": "3.0.1",
   "description": "Shields.io badge library",
   "keywords": [
     "GitHub",

--- a/badge-maker/package.json
+++ b/badge-maker/package.json
@@ -35,6 +35,7 @@
   },
   "dependencies": {
     "anafanafo": "^1.0.0",
+    "camelcase": "^5.3.1",
     "is-css-color": "^1.0.0"
   },
   "scripts": {

